### DISCUSSION TITLE
How can  study more on how to contribute and debug the MathJS library

### DISCUSSION THREAD

**Discussion by nkumawat34** [josdejong/mathjs#3305] [Q&A]
I want to learn more from the math.js documentation on how to contribute to this library. How can I start debugging it as a beginner?

**Comment by dvd101x** [comment]
Hi Neeraj,

You could start by reading the main [readme](https://github.com/josdejong/mathjs/blob/develop/README.md) file and follow the [develop instructions](https://github.com/josdejong/mathjs/blob/develop/README.md#develop) if you haven't done that yet.

After that you could search for Issues and Discussions with the tags **help wanted** find something that interests you, asking if you could work on the issue and make a PR.

I would recommend to start with something simple that interests you the most.
